tasks/opsep/update-l2pao-to-aliased-l1pao/SignFromJson.s.sol

```diff
@@ -0,0 +1,188 @@
+import {OptimismMintableERC20Factory} from "@eth-optimism-bedrock/src/universal/OptimismMintableERC20Factory.sol";
+import {L1ERC721Bridge} from "@eth-optimism-bedrock/src/L1/L1ERC721Bridge.sol";
+import {Predeploys} from "@eth-optimism-bedrock/src/libraries/Predeploys.sol";
+import {ISemver} from "@eth-optimism-bedrock/src/universal/ISemver.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {EIP1967Helper} from "@eth-optimism-bedrock/test/mocks/EIP1967Helper.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+
+// Interface used to read various data from contracts. This is an aggregation of methods from
+// various protocol contracts for simplicity, and does not map to the full ABI of any single contract.
+interface IFetcher {
+    function overhead() external returns (uint256); // SystemConfig
+    function scalar() external returns (uint256); // SystemConfig
+    function guardian() external returns (address); // SuperchainConfig
+    function L2_BLOCK_TIME() external returns (uint256); // L2OutputOracle
+    function SUBMISSION_INTERVAL() external returns (uint256); // L2OutputOracle
+    function FINALIZATION_PERIOD_SECONDS() external returns (uint256); // L2OutputOracle
+    function startingTimestamp() external returns (uint256); // L2OutputOracle
+    function startingBlockNumber() external returns (uint256); // L2OutputOracle
+    function owner() external returns (address); // ProtocolVersions
+    function required() external returns (uint256); // ProtocolVersions
+    function recommended() external returns (uint256); // ProtocolVersions
+}
+
+interface IProxyAdmin {
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    function owner() external view returns (address);
+
+    function transferOwnership(address newOwner) external;
+}
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    string constant l1ChainName = "sepolia";
+    string constant l2ChainName = "op";
+
+    // Safe contract for this task.
+    GnosisSafe l2paoSafe = GnosisSafe(payable(0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB));
+    address constant l2PaoSafeEOA = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301;
+
+    // Contracts we need to check, which are not in the superchain registry
+    IProxyAdmin l2pa = IProxyAdmin(0x4200000000000000000000000000000000000018);
+    // Aliased L1PAO
+    address constant aliasedL1PAO = 0x2FC3ffc903729a0f03966b917003800B145F67F3;
+    address constant unaliasedL1PAO = 0x1Eb2fFc903729a0F03966B917003800b145F56E2;
+    uint160 private constant ALIASING_OFFSET = uint160(0x1111000000000000000000000000000000001111);
```

Comment:
Let's minimize hardcoding as much as we can—this will make it less risky when we port to the mainnet version:
- The l2PaoSafeEOA can be fetched from the safe
- 0x4200000000000000000000000000000000000018 can be replaced with `Predeploys.ProxyAdmin` (import from the monorepo dep)
- Aliased addresses can be computed using the `AddressAliasHelper` library (also from the monorepo dep)

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/204/commits/5055a9049e26cf7cd4c847be600d41042295e377